examples/platform/linux/AppMain.cpp

```diff
@@ -1017,6 +1017,9 @@ void ChipLinuxAppMainLoop(AppMainLoopImplementation * impl)
     sigaction(SIGTERM, &sa, nullptr);
 #endif
 
+    // This message is used as a symptom for when the application process has started.
```

Comment:
I would change the "symptom" to something else because it sounds like "something is not right and we are detecting symptoms". Also indicate were to look at (some method like `App.waitForApplicationUp()` or something else).